doc/explanation/networks.md

```diff
@@ -28,11 +28,11 @@ You can configure the network device in one of the following ways:
 
         lxc config device add <instance_name> eth0 nic nictype=bridged parent=br0
 - {doc}`Create a managed network </howto/network_create>` and add it as a network device to your instance.
-  With this method, LXD has all required information about the configured network, and you only need to specify the network name when adding the device:
+  With this method, LXD has all required information about the configured network, and you can directly attach it to your instance as a device:
 
-        lxc config device add <instance_name> <device_name> nic network=<network_name>
+        lxc network attach <network_name> <instance_name>
```

Comment:
I think we should recommend using `<device_name>` and give an example of using the `<device_name>` as `eth0`. The reason being that LXD images are built to perform IP autoconfiguration only on the `eth0` interface, and by default doing `lxc network attach <network_name> <instance_name>` LXD will attach the device named after the network_name, and so won't get IP auto configuration by default.

Reply:
This is a good example of someone who has likely got confused about `lxc network attach` and added a NIC named after the parent network:

https://discuss.linuxcontainers.org/t/error-failed-start-validation-for-device-enp3s0f0-instance-dns-name-net17-nicole-munoz-marketing-already-used-on-network/15586/26?u=tomp

Reply:
Thanks for the explanation! Updated.